deployment/helm/values.yaml

```diff
@@ -98,8 +100,12 @@ edgex:
         tag: "2.3.0"
         pullPolicy: IfNotPresent
       httpexport:
-        repository: nexus3.edgexfoundry.org:10004/app-service-configurable
-        tag: "latest"
+        repository: edgexfoundry/app-service-configurable
+        tag: "2.3.0"
+        pullPolicy: IfNotPresent
+      externalmqtttrigger:
+        repository: edgexfoundry/app-service-configurable
+        tag: "2.3.0"
```

Comment:
If nats is used then the locally built image must be referenced. Same for all the services

Reply:
Yes, got another issue for that:

https://github.com/edgexfoundry/edgex-examples/issues/157